doc/help/elektra-glossary.md

```diff
@@ -102,6 +102,34 @@
 - [Metadata](elektra-metadata.md):
   Allows us to describe configuration settings.
 
+- **Namespace Root Key**:
+  A namespace root key, is the root key of a namespace.
+  It consists only of a namespace, but no other parts.
+  e.g.: `system:/`, `user:/`, `/`
+
+- **Child** (of key K):
+  A key whose name starts the same as that of K, but has exactly one extra part.
+
+- **Parent** (of key K):
+  A key P where K is a child of P.
+  A namespace root key has no parent, every other key has exactly one parent.
+
+- **Descendant** (of key K):
```

Comment:
This means we need to change all occurrences (inclusive the API) of "below"

Reply:
It is not only a lot of work for you but also for me (lecture materials etc.). So if there isn't a good reason, we definitely shouldn't do it.

Reply:
> This means we need to change all occurrences (inclusive the API) of "below"

No it doesn't. It means, we to explain "below" as well. I don't see why we couldn't have synonyms. However, if you want exactly one term for everything (*) then "below" is clearly a worse choice, since AFAIK there is no noun form of "below".

(*) Having exactly one term can be good for precise specifications and we if we have synonyms we should decide on a "specification term" for everything. But specifications can be quite tedious to read. Most forms of documentation become better, if they are written in a less formal prose style. In that case it helps, if you have some synonyms to use.